test/fail_compilation/test17284.d

```diff
@@ -1,8 +1,9 @@
 TEST_OUTPUT:
 ---
-fail_compilation/test17284.d(1): Error: no identifier for declarator `TEST_OUTPUT`
```

Comment:
This test is broken, the first line needs a /* to put `TEST_OUTPUT:` in a comment

Reply:
I didn't care to make this about fixing other tests.

Reply:
You're updating the test anyway, why not do it properly?

Reply:
The test is *about* the error generated by forgetting the /*.

Reply:
No, this is what the test is supposed to look like: https://github.com/WalterBright/dmd/blob/36eb3834efa4e792221fffb3badd0fff5d3323a7/test/fail_compilation/test17284.d

Reply:
The PR changes what it *is*, which is missing the opening /*, not what it is *supposed* to be. Fixing what tests are supposed to be is not relevant to this PR.

Reply:
Ok, https://github.com/dlang/dmd/pull/13967